Updated to latest BattleSnake API, found here: https://docs.battlesna…

KenWright22 opened the pull request:
Updated to latest BattleSnake API for the core objects. Ref: https://docs.battlesnake.com/references/api
- Added Hazards property for Board.
- Added Ruleset property for Game.
- Added Latency and Squad for Snake.
- Some whitespace changes as well (removing excess spaces)

Commit:
Updated to latest BattleSnake API, found here: https://docs.battlesnake.com/references/api
- Added Hazards property for Board.
- Added Ruleset property for Game.
- Added Latency and Squad for Snake.

neistow commented on Starter.Core/Game.cs:
```diff
@@ -14,7 +14,15 @@ public class Game
+        /// Information about the ruleset being used to run this game.
+        /// Example: Example: {"name": "standard", "version": "v1.2.3"}
+        /// </summary>
+        public string Ruleset { get; set; }
```
It would be better if Ruleset is represented by a class, not a string

KenWright22 replied:
Agreed. I have updated the code and I believe it is now part of the PR.

Commit:
Added Ruleset class, and updated reference to it from Game class.

Commit:
Improved the the format of some comments.
- Had the word "Example" repeated in Game.cs
- Modified the examples in Ruleset.cs to follow the format elsewhere in this project.
